packages/cli/test/integration/generators/relation.belongs-to.integration.js

```diff
@@ -265,7 +238,7 @@ describe('lb4 relation', function() {
           .withPrompts(multiItemPrompt);
       });
 
-      it('relation name should be ' + defaultRelationName[i], async () => {
+      it('relation name should be customerPK', async () => {
```

Comment:
hmm, I am not very familiar with the test here, is it necessary to replace the computed relation name `defaultRelationName[i]` with a fixed one(`customerPK`)?

Reply:
The default values are tested in the test `'generates model relation with default values'`. So I changed it to test `relationName` for 2 reasons:
1. The CLI seems to allow users to set up any relation name:
```
? Source property name for the relation getter: randomm
```
this would be 
```
this.cat = this.createBelongsToAccessorFor('randomm', catRepositoryGetter); // this doesn't seem right to me
```
( I think we actually have a bug here...)

2. Lineup with hasMany as it has this test. 


But as I mentioned above. I think our code has bugs, and this test should also check `repository.ts` as well to make sure it uses the correct relation name. But I decide to remove the redundant tests and move on the inclusion CLI task first, them to see if it _does_ have bugs